src/include/sof/ssp.h

```diff
@@ -236,6 +236,7 @@ struct ssp_pdata {
 	uint32_t psp;
 	spinlock_t lock;
 	uint32_t state[2];		/* SSP_STATE_ for each direction */
+	uint32_t lbm;
```

Comment:
This is platform data, so it cant hold any "status". It can however store a flag that can be used to do a one time configure of the SSP hardware. Please also use a more descriptive name for this, what does it do ?

Reply:
Will use register value.